src/nupic/regions/knn_anomaly_classifier_region.capnp

```diff
@@ -0,0 +1,63 @@
+@0xc8ac55fe2407eceb;
+
+using import "/nupic/regions/knn_classifier_region.capnp".KNNClassifierRegionProto;
+
+# Next ID: 5
+struct ClassificationRecord {
+    rowid @0 :Int32;
+    anomalyScore @1 :Float32;
+    anomalyVector @2 :List(Int32);
+    anomalyLabel @3 :List(Text);
+    setByUser @4 :Bool;
+}
+
+# Net ID: 16
+struct KNNAnomalyClassifierRegionProto {
+    version @0 :Int32;
+    maxLabelOutputs @1 :Int32;
+    activeColumnCount @2 :Int32;
+    prevPredictedColumns @3 :List(Int32);
+    anomalyVectorLength @4 :Int32;
+    classificationMaxDist @5 :Float32;
+    iteration @6 :Int32;
+    trainRecords @7 :Int32;
+    anomalyThreshold @8 :Float32;
+    cacheSize @9 :Int32;
+    classificationVectorType @10 :Int32;
+    knnclassifierArgs @11 :KNNClassifierArgsProto;
+    knnclassifier @12 :KNNClassifierRegionProto;
+    labelResults @13 :List(Text);
+    savedCategories @14 :List(Text);
+    recordsCache @15 :List(ClassificationRecord);
+
+    # Next ID: 26
+    struct KNNClassifierArgsProto {
+               maxCategoryCount @0 :Int32;
```

Comment:
Indentation is weird in this file. We have been using 2-spaces per level in other files.